app/models/user.rb

```diff
@@ -1,12 +1,20 @@
 class User < ApplicationRecord
-  has_many :comments, foreign_key: 'user_id'
-  has_many :posts, foreign_key: 'author_id', primary_key: 'id'
-  has_many :likes, foreign_key: 'user_id'
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable, :confirmable
```

Comment:
[REQUIRED] Could you remove the repetition for the devise features? For example, I can see `:database_authenticatable` repeated twice among others